src/utils/listingUtils.js

```diff
@@ -1,44 +1,101 @@
-export const NEEDS_SHEET_NAME = "Form Responses 1"
+export const NEEDS_SHEET_NAME = "Public needs responses"
 
 export const NEEDS_SHEET_COLUMN_INDICES = {
   createdAt: 0,
+  onBehalfOf: 1,
   name: 2,
   pronouns: 3,
   location: 4,
   preferredContactMethod: 5,
   contact: 6,
+  interestedInConnecting: 7,
   neighborhoodName: 8,
-  public: 9,
-  isFirstNeed: 11,
 
   // FINANCIAL NEED
-  isFinancialNeed: 12,
-  financial_needFrequency: 13,
-  financial_needTiming: 14,
-  financial_minFundingNeed: 17,
-  financial_maxFundingNeed: 18,
-  financial_fundingMethod: 19,
+  isFinancialNeed: 9,
+  financial_needFrequency: 10,
+  financial_needTiming: 11,
+  financial_minFundingNeed: 14,
+  financial_maxFundingNeed: 15,
+  financial_fundingMethod: 16,
 
   // SUPPIES/ERRANDS NEED
-  isSuppliesNeed: 20,
-  supplies_needFrequency: 21,
-  supplies_needTiming: 22,
-  supplies_details: 23,
-  supplies_neighborhood: 24,
-  supplies_store: 25,
-  supplies_shoppingList: 26,
+  supplies: {
+    id: 18,
+    meta: {
+      needFrequency: 19,
+      needTiming: 20,
+      costCoverage: 21,
+      details: 22,
+      neighborhood: 23,
+      store: 24,
+      shoppingList: 25,
+    },
+  },
 
   // TRANSPORTATION
-  isTransportationNeed: 27,
-  transportation_needFrequency: 28,
-  transporation_needTiming: 29,
-  transportation_details: 30,
-  transportation_neighborhood: 31,
-  transportation_comments: 32,
+  isTransportationNeed: 26,
+  transportation_needFrequency: 27,
+  transporation_needTiming: 28,
+  transportation_details: 29,
+  transportation_neighborhood: 30,
+  transportation_comments: 31,
+
+  // HOUSING
+  housing: {
+    id: 32,
+    meta: {
+      whatKindOfSpace: 33,
+      needTiming: 34,
+      needTransportationToHousing: 35,
+      bedPreferences: 36,
+      petAllergies: 37,
+      dietaryRestrictions: 38,
+      mobility: 39,
+      neighborhood: 40,
+      ageGenderPreference: 41,
+      comments: 42,
+      storageQuantity: 43,
+      climateControlled: 44,
+      storageNeedTiming: 45,
+      storageComments: 46,
+    },
+  },
+
+  childcarePetcare: {
+    id: 47,
+    meta: {
+      ageAndNumberOfKids: 48,
+      typeOfRequest: 49,
+      typeOfActivities: 50,
+      needFrequency: 51,
+      needTiming: 52,
+      comments: 53,
+      kindOfPets: 54,
+      supportType: 55,
+      petNeedFrequency: 56,
+      petNeedTiming: 57,
+      petComments: 58,
+    },
+  },
+
+  emotionalSupport: {
+    id: 59,
+    meta: {
+      supportType: 60,
+      resourceSupport: 61,
+      resourceSupportDetails: 62,
```

Comment:
Ah @esetnik sorry one thing I missed! Resource support is its own category, after emotional support.

Reply:
Ok so emotional support will only have one item with the support type?

Reply:
Awesome, @esetnik you're the best

Reply:
Love the new structure, too